core/src/main/scala/com/banno/kafka/consumer/SeekTo.scala

```diff
@@ -147,6 +156,9 @@ object SeekTo {
 
   def end: SeekTo = Impl(List.empty, FinalFallback.end)
 
+  def fail(throwable: Throwable): SeekTo =
```

Comment:
Is it worth having two fail methods?

```scala
final case class PathologicalSeekTo() extends RuntimeException("You shall not pass!")

def failWith(throwable: Throwable): SeekTo =
  Impl(List.empty, FinalFallback.fail(throwable))

def fail: SeekTo =
  failWith(PathologicalSeekTo())
```

Reply:
I avoided this to avoid having to pick a name, but you've forced my hand 😄